website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -76,6 +76,28 @@ need to rotate keys.
 
 New Nomad clusters will use RSA by default and are not affected.
 
+#### CPU Fingerprinting Changes
+
+Starting in Nomad 1.7, Nomad clients improve the accuracy of detected CPU
+performance metrics. The fingerprinter now takes into account heterogeneous core
+types on applicable processors. In addition, Nomad will attempt to detect and
+use the base frequency of the processor rather than the turbo frequency when
+calculating the total availble CPU bandwidth. The net result of these behaviors
```

Comment:
```suggestion
calculating the total available CPU bandwidth. The net result of these behaviors
```

Reply:
I need to install spell check .... :slightly_frowning_face: